### ISSUE TITLE
Questions about performance with other partially overlapping data

### ISSUE THREAD

**Issue by CorgiBoyG** [haoyu94/RoITr#15]
Thanks for making such a great work. Recently, I tried to train RegFormer on ModelNet, ModelLoNet, Kitti, but the results are not so good. For example, I trained on ModelNet with no modifications to the network, optimizer, and other parameters, and immediately after training for 150 epochs, I evaluated it, and the final _rre_ calculated for the validation set was 42. Could you please tell me what modifications I need to make, e.g., parameters, etc.? 
Can you please provide some suggestions to help me, it is very important for me, thanks! Thanks again for the superior work you have done.
Sincerely looking forward to your reply!

**Comment by k794102577** [comment]
I fixed the kitti problem, but not the modelnet problem

**Comment by k794102577** [comment]
For modelnet, did you test it on a clean, complete point cloud, and how did it work

**Comment by CorgiBoyG** [comment]
> For modelnet, did you test it on a clean, complete point cloud, and how did it work



> I fixed the kitti problem, but not the modelnet problem

How did you resolve this and could you offer some advice?

**Comment by CorgiBoyG** [comment]
> I fixed the kitti problem, but not the modelnet problem

How did you resolve this and can you offer some advice?

**Comment by CorgiBoyG** [comment]
> For modelnet, did you test it on a clean, complete point cloud, and how did it work

I have only tried modelnet under partial overlap, following the rpmnet protocol for partial overlap data generation. Found it didn't work well, rre was only 42.

**Comment by k794102577** [comment]
My suggestion is to adapt the entire framework to geotransformer parameters and then load kitti data using predator preprocessed code,it is not difficult

**Comment by k794102577** [comment]
For the modelnet dataset, I can also trainning smoothly

**Comment by CorgiBoyG** [comment]
> For the modelnet dataset, I can also trainning smoothly

Okay, could you tell me how you adjusted it, please. I want to use roitr as a comparison algorithm, not an improved optimization based on Roitr. If it's convenient for you, could you guide me in detail what I should change. I tried to change the parameters of roitr yesterday against GeoTransformer, but the training remains the same!

**Comment by CorgiBoyG** [comment]
> For the modelnet dataset, I can also trainning smoothly

Excuse me. Can you help me please, I'm only using it for comparison algorithm experiments, not plagiarizing. This is very important to me, thanks. Looking forward to your reply.

**Comment by k794102577** [comment]
First of all, we also used the preprocessing code of predator for the modelnet data set, which was verified and found that the framework could be trained smoothly under crop, jitter and clean conditions, and the test results were found to be normal. After the training, I will upload the screenshot

**Comment by CorgiBoyG** [comment]
> First of all, we also used the preprocessing code of predator for the modelnet data set, which was verified and found that the framework could be trained smoothly under crop, jitter and clean conditions, and the test results were found to be normal. After the training, I will upload the screenshot

May I ask how your subsequent experiments have worked out and if you are getting good performance?

**Comment by CorgiBoyG** [comment]
> My suggestion is to adapt the entire framework to geotransformer parameters and then load kitti data using predator preprocessed code,it is not difficult

If it is convenient for you, is it possible to provide the modified code? I promise to use it only for experimental comparison testing.

**Comment by k794102577** [comment]
You can comment out the data enhancement code without affecting the final comparison test of the experiment
